policyengine_us/parameters/gov/states/ct/tax/income/credits/property_tax/amount/separate.yaml

```diff
@@ -0,0 +1,46 @@
+description: Connecticut provides separate filers a property tax credit for this fraction of taxes, depending on their adjusted gross income.
+metadata:
+  type: single_amount 
+  threshold_unit: currency-USD
+  amount_unit: /1
+  label: Connecticut Property tax credit for separate filers
+  reference:
+  - title: Connecticut General Statutes, Chapter 229, Sec. 12-704c
+    href: https://www.cga.ct.gov/current/pub/chap_229.htm#sec_12-704c
+  - title: 2022 Form CT-1040 Connecticut Resident Income Tax Return Instructions
+    href: https://portal.ct.gov/-/media/DRS/Forms/2022/Income/2022-CT-1040-Instructions_1222.pdf#page=26
+  - title: 2021 Form CT-1040 Connecticut Resident Income Tax Return Instructions
+    href: https://portal.ct.gov/-/media/DRS/Forms/2021/Income/CT-1040-Online-Booklet_1221.pdf#page=30 
+brackets:
+  - threshold: 
```

Comment:
we need to convert these according to the law - make a reduction folder with files called increment, amount and start - see ct addback as an example in the model

Reply:
we need to model the structure by the law: https://www.cga.ct.gov/current/pub/chap_229.htm#sec_12-704c